roles/ceph-osd/tasks/system_tuning.yml

```diff
@@ -32,6 +32,32 @@
     when: ursula_os == 'rhel'
   when: ceph.disable_transparent_hugepage
 
+- name: create ursula tuned profile dir
+  file: path=/etc/tuned/ursula state=directory
+  when: ursula_os == 'rhel'
+
+# tune disk read-ahead for ceph
+# if there are more tunings in future,
+# we should move the common part(main section) into common role
+- name: set disk read-ahead for ceph
+  ini_file:
```

Comment:
Let me explicate `tuned doesn't support more than one include`
Below code doesn't work, because only one include could be used.
```
[main]
include=ceph.conf
include=neutron.conf
```
Below code doesn't work neither, tuned says no profile named `ceph.conf, neutron.conf`
```
[main]
include=ceph.conf, neutron.conf
```